pkg/controllers/node/termination/controller.go

```diff
@@ -83,7 +83,7 @@ func (c *Controller) Finalize(ctx context.Context, node *v1.Node) (reconcile.Res
 		return reconcile.Result{}, fmt.Errorf("deleting nodeclaims, %w", err)
```

Comment:
Is it worth adding a printer-column over the disruption taint in this PR?

Reply:
It potentially could just be printed as part of the `-o wide` output

Reply:
I'd prefer to look into it as a follow-up if that's cool 👍